lib/language_pack/installers/ruby_installer.rb

```diff
@@ -0,0 +1,30 @@
+require "language_pack/shell_helpers"
+module LanguagePack::Installers; end
+
+module LanguagePack::Installers::RubyInstaller
+  include LanguagePack::ShellHelpers
+
+  DEFAULT_BIN_DIR = "bin"
+
+  def self.installer(ruby_version)
+    if ruby_version.rbx?
+      LanguagePack::Installers::RbxInstaller
+    else
+      LanguagePack::Installers::MriInstaller
```

Comment:
For naming the MRI handles JRuby as well. Maybe it makes more sense to call it something more generic? `HerokuRubyInstaller` (or other), the differentiation is between what we support via our bucket and what we don't versus the interpreter?